convert_types.go

```diff
@@ -0,0 +1,8 @@
+package datadog
+
+func Int(val interface{}) *int {
```

Comment:
:+1: I tend to agree. I'd borrowed the implementation from `aws/aws-sdk-go` (which I meant to call out in the PR description), but I think explicit is better.